### DISCUSSION TITLE
casperClient.putDeploy and casperClient.nodeClient.deploy.deploy methods always return pending promise

### DISCUSSION THREAD

**Discussion by LookerClocker** [casper-ecosystem/casper-client-sdk#333] [Q&A]
Hi guys! 

Im trying to send deploy using casperClient. The both methods do their job and works well. But I've faced an issue (or its maybe not an issue) that  Promise<string> which returns casperClient.putDeploy() always has pending status. Due that I can't handle some actions in my codebase. 
Basically here the example:
```
 const signedDeploy = await casperClient.putDeploy(result.unwrap())
        console.log(signedDeploy); 
```

For some reason console.log(signedDeploy);  is unreachable code. 
Could you please explain me how to handle this issue? Many thanks and appreciate your answer :)

**Comment by hoffmannjan** [comment]
@LookerClocker - seems like it wasn't problem on our side so we can close it right?
